src/capi_frontend/capi.cpp

```diff
@@ -1045,7 +1051,7 @@ DLL_PUBLIC OVMS_Status* OVMS_Inference(OVMS_Server* serverPtr, OVMS_InferenceReq
         status = pipelinePtr->execute(executionContext);
         // INCREMENT_IF_ENABLED(pipelinePtr->getMetricReporter().getInferRequestMetric(executionContext, status.ok()));
     } else {
-        status = modelInstance->infer(req, res.get(), modelInstanceUnloadGuard);
+        status = ovms::infer(*modelInstance, req, res.get(), modelInstanceUnloadGuard);
```

Comment:
do we really need to refer to ovms namespace? we are in ovms namespace anyway

Reply:
In capi.cpp file we are not.